### PR TITLE
Event view

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**
Add organizedBy to the meeting
Bunch of small updates

### CODE REVIEW

File: libs/event/src/lib/layout/view/view.component.html [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
+        <h6>{{ event.type }} done</h6>
+        <pre class="mat-body-1">{{ event | eventRange }}</pre>
+      </ng-container>
+
+    </ng-container>
   </header>
-  <section>
-    <ng-content></ng-content>
-  </section>
+  <ng-content></ng-content>
```
Comment (phpgeekfr) [inline libs/event/src/lib/layout/view/view.component.html:26]: typo

File: libs/user/src/lib/user/+state/user.store.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import { Injectable } from "@angular/core";
 import { StoreConfig, EntityStore, EntityState } from "@datorama/akita";
-import { OrganizationMember } from "./user.model";
```
Comment (phpgeekfr) [inline libs/user/src/lib/user/+state/user.store.ts:3]: can you remove OrganizationMember from the whole app  ?

Comment (GrandSchtroumpf) [reply libs/user/src/lib/user/+state/user.store.ts:3]: Clélia is going to do it. I think it worth an whole issue.

Comment (phpgeekfr) [reply libs/user/src/lib/user/+state/user.store.ts:3]: Should I create the issue ?

**Pull request merged**